packages/ncids-css/scss/components/nci-header/internal/_header.scss

```diff
@@ -0,0 +1,84 @@
+
+// NCI LOGO STYLES
+.usa-logo--nci-logo {
+	display:block;
+	width:340px;
+	height:38px;
+	//tablet 
+  @include at-media-max($theme-header-min-width) {
+  }
+	//desktop
+  @include at-media($theme-header-min-width) {
+  }
+  //mobile
+  @include at-media-max($theme-modal-default-max-width) {
+  }
+  a {
+    color: color("ink");
+    text-decoration: none;
+  }
+}
+
+// NCI HEADER EXTENDED
+.nci-header-extended {
+	border:0;
+
+	.nci-header--nci-navbar {
+		@include border-box-sizing;
+		border:0;
```

Comment:
nit: how did the linter not pick up spaces in the rules.  it's peppered throughout